src/BSON/Javascript.stub.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/** @generate-function-entries */
+
+namespace MongoDB\BSON
+{
+    final class Javascript
+    {
+#if PHP_VERSION_ID >= 80000
+        final public function __construct(string $code, array|object|null $scope = null) {}
```

Comment:
Can the nullable type hint (`?` prefix) be used here or is `|null` required when using type unions?

Reply:
PHP requires `|null` for nullable unions. This came up in the discussion for [nullable intersection types](https://wiki.php.net/rfc/nullable_intersection_types), which would essentially be a union of `null` and the intersection type.